### PR TITLE
add tokyo_night theme

### PR DESCRIPTION
**Pull request by AviAvinav**
### Description

I have added the Tokyo Night Theme, based on a famous theme from vscode.

Official Tokyo Night Theme for vscode repo: https://github.com/enkia/tokyo-night-vscode-theme.

This is how it looks:

Also, thanks for making monkeytype, has been quite helpful to me.

### CODE REVIEW

**Comment by Bruception** [comment]
Please include additional screenshots as noted in the theme contribution guide

**Comment by AviAvinav** [comment]
Added another screenshot, couldn't find what exactly I had to include in the screenshots, so, sorry if it is wrong. May you please review it again.

File: frontend/static/themes/tokyo_night.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:root {
+  --bg-color: #1a1b26;
+  --main-color: #e0af68;
+  --caret-color: #e0af68;
+  --sub-color: #414868;
+  --sub-alt-color: #414868;
```
Comment (Miodec) [inline frontend/static/themes/tokyo_night.css:6]: Duplicate color. Will probably cause issues somehwere. Please update.

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by Miodec** [comment]
I just realised - looks too similar to moonlight theme. Sorry.

**Comment by AviAvinav** [comment]
Ok, no problem

**Pull request closed without merging**